topics/galaxy-interface/tutorials/group-tags/tutorial.md

```diff
@@ -51,13 +54,16 @@ testing see the [Reference-based RNA-Seq data analysis]({{ site.baseurl }}/topic
 
 # Setting group tags using the apply rules tool
 
-There are three ways to set group tags:
-  - Using the rule builder / apply_rules tool
+There are several ways to set group tags:
+  - Using the Rule Based Uploader
   - Using the "Tag elements from file" tool
+  - Using the "Apply Rules" tool
   - Manually adding dataset tags with the prefix `group:`
```

Comment:
Actually there is one more: from the history one can `For all selected ...` `->` `Build collection from rule`.

Maybe it would be good to reformulate the tutorial such that the upload, apply rules, build collection paths are covered. The only difference seems to be that the `URL` column needs to (can be) be  defined only in the upload option.

Reply:
Ah forgot about that option. We really should cover each, you're right, that sounds like a good improvement! You're probably more familiar with group tags, would you want to open a 'help wanted' issue for updating the tutorial for the other use-cases that are important in your view?

Reply:
Moved to #2527